Makefile

```diff
@@ -1,6 +1,27 @@
+UNAME_S ?= $(shell uname -s)
+FFMPEG_PREFIX ?= $(shell pwd)/ffmpeg/$(UNAME_S)-$(shell uname -m)
+FFMPEG_OPTS ?= --prefix=$(FFMPEG_PREFIX) \
+               --enable-static \
+               --disable-shared \
+               --disable-programs \
+               --disable-doc \
+               --disable-everything \
+               --enable-decoder=h264 \
+               --enable-decoder=hevc \
+               --enable-swscale
+ifeq ($(UNAME_S),Linux)
+    CGO_LDFLAGS := "-L$(FFMPEG_PREFIX)/lib -l:libjpeg.a"
```

Comment:
This assumes `libjpeg.a` is available on the machine (we are in a canon container).